src/main/resources/validationrules/civil/civil-ctsc-mapping.drl

```diff
@@ -100,13 +135,46 @@ then
       .build());
       logMsg("Rule : v11_civil_ctsc_ctsc_org_role");
 end;
+
+/*
+ * CIVIL admin "ctsc" v22 Org role mapping.
+ */
+
+rule "v2_2_civil_team_lead_ctsc_org_role"
```

Comment:
Can you merge the two new `v2_2_civil_admin_ctsc_org_role` and `v2_2_civil_team_lead_ctsc_org_role` into one `v2_2_civil_ctsc_org_role` rule which works for both roles 9 & 10 as the properties appear to be the same.

Reply:
Resolved in 752f92c0f006d0c84ebdf3ec2253513e97850ba2.